app.js

```diff
@@ -1,11 +1,82 @@
 /* Imports */
 
 /* Get DOM Elements */
+const cityArticle = document.getElementById('city-article');
+
+const nameInput = document.getElementById('name-input');
+const nameDisplay = document.getElementById('name-display');
+const climateSelect = document.getElementById('climate-select');
+const climateDisplay = document.getElementById('climate-image');
+const architectureSelect = document.getElementById('architecture-select');
+const architectureDisplay = document.getElementById('architecture-image');
+const attractionInput = document.getElementById('attraction-input');
+const addButton = document.getElementById('add-button');
+const attractionList = document.getElementById('attraction-list');
 
 /* State */
+let city = {
+    name: 'Cool Town',
+    climate: 'desert',
+    architecture: 'modern',
+    attractions: [],
+};
+
+/* Helper Functions */
+function pageLoad() {
+    displayControls();
+    displayCity();
+    displayAttractions();
+}
 
 /* Events */
+nameInput.addEventListener('input', () => {
+    city.name = nameInput.value;
+    pageLoad();
+});
+
+climateSelect.addEventListener('change', () => {
+    city.climate = climateSelect.value;
+    pageLoad();
+});
+
+architectureSelect.addEventListener('change', () => {
+    city.architecture = architectureSelect.value;
+    pageLoad();
```

Comment:
It might break something, I'm not sure, but you might be able to get away with just running ```displayCity()``` rather than loading the whole page